tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -166,7 +166,7 @@ try {
       if (!compression) {
          // Get the compression of this RNTuple and use it as the output compression.
          // We currently assume all column ranges have the same compression, so we just peek at the first one.
-         source->Attach();
+         source->Attach(RNTupleSerializer::EDescriptorDeserializeMode::kRaw);
```

Comment:
In `RNTupleMerger::Merge`, we attach the items of `sources` in the default mode, but sometimes we have the first source attached in raw mode. That probably has no practical relevance but I would find it better to consistently decide for one attach mode.

Reply:
I guess we should always attach in `Raw` mode, as it's the one that does the least amount of work..